src/main/java/uk/gov/justice/hmpps/prison/api/model/OffenderKeyDates.java

```diff
@@ -61,4 +61,7 @@ public class OffenderKeyDates {
 
     @Schema(required = true, description = "Sentence length in the format 00 years/00 months/00 days.", example = "11/00/00")
     private String sentenceLength;
+
+    @Schema(description = "Judicially imposed length in the format 00 years/00 months/00 days. Will default to 'sentenceLength'", example = "11/00/00")
+    private String judiciallyImposedSentenceLength;
```

Comment:
Is there any reason we couldn't make this a small object even if we think people won't use it ?
{
 years: Int
months: Int
days: Int
}

Or whatever?
Maybe not worth it?

Reply:
Yea I think that would be better. We're just matching what nomis is doing here. In CRD its modelled as a java Period, which works really well. Hopefully these fields will be going away soon, if not I'll look at changing it.

Reply:
Yes!